src/Features/CSharp/Portable/CodeFixes/AddImport/CSharpAddImportCodeFixProvider.cs

```diff
@@ -701,11 +703,11 @@ private bool IsViablePropertyOrField(ISymbol propertyOrField, SyntaxNode express
                 return false;
             }
 
-            var comparisonType = this.IgnoreCase
-                ? StringComparison.OrdinalIgnoreCase
-                : StringComparison.Ordinal;
+            var comparer = this.IgnoreCase
+                ? CaseInsensitiveComparison.Comparer
+                : StringComparer.Ordinal;
 
-            return string.Compare(propertyOrField.ContainingType.Name, leftName.Identifier.Text, comparisonType) == 0;
+            return comparer.Compare(propertyOrField.ContainingType.Name, leftName.Identifier.Text) == 0;
         }
```

Comment:
I assume this goes away with your other change to avoid the check to `IgnoreCase`?